### DISCUSSION TITLE
Wiring the seven-segment hex display

### DISCUSSION THREAD

**Discussion by carlos54vidal** [hneemann/Digital#1064] [Q&A] [answered]
Hello,
I need help on how to use the seven-segment hex display, since there is a unique input for the 4 bits.

**Comment by hneemann** [comment, marked as answer]
The simulator uses bus wirese that can contain more than one bit.
You must use a splitter/merger to split the bus into multiple bits. 
![seven](https://user-images.githubusercontent.com/7190563/205139974-07d0ec6d-fa88-4773-b10a-1eb4e82fbec5.png)

**Comment by carlos54vidal** [comment]
Thanks
________________________________
